### PR TITLE
Review

### PR DESCRIPTION
**Pull request by carriewright11**
Just putting small changes here. Will put conceptional review thoughts on basecamp with Candace's review.

### CODE REVIEW

**Comment by carriewright11** [comment]
Really great job! I think this is very clear :)

File: 01-101.Rmd [hunk 1]
```diff
@@ -45,14 +45,15 @@ For example:
...
-If one of your collaborators requires access to the Fred Hutch network you can submit a [non-employee action form](https://centernet.fredhutch.org/cn/f/hr/lcex/non-employee-action-form.html). Non-employee is a generic administrative term for affiliates, students, contractors, etc.
+If one of your collaborators requires access to the Fred Hutch network you can submit a [non-employee action form](https://centernet.fredhutch.org/cn/f/hr/lcex/non-employee-action-form.html). "Non-employee" is a generic administrative term for affiliates, students, contractors, etc.
 </div>
 
 ## Connect to a PI Account {#pi-account}
 
 Your HutchNet ID must be associated with a PI cluster "account". The Scientific Computing Team (SciComp) tries to set users up with a connection to a PI account before they need it, but this is not automatic!  To ensure that you have been set up to use the cluster, please follow the following steps:
 
-1. Go to the [SciComp Self-Service Portal](https://scicomp-self-service.fredhutch.org/)
+1. Log into the Hutch VPN if needed (log in using Hutch email and CenterNet password)
```
Comment (carriewright11) [inline 01-101.Rmd:55]: might want to edit this like you have it later

**Comment by avahoffman** [comment]
> I wonder if it would be worth talking about security... maybe you do later

I think this is a great idea! Maybe for 201? I can also bring it up in our regular meeting and see if they think it would be appropriate for 101

**Comment by carriewright11** [comment]
> > I wonder if it would be worth talking about security... maybe you do later
> 
> I think this is a great idea! Maybe for 201? I can also bring it up in our regular meeting and see if they think it would be appropriate for 101

Yeah, I think that is a good plan. I was mostly thinking just a short disclaimer or something.

**Comment by carriewright11** [comment]
> > I wonder if it would be worth talking about security... maybe you do later
> 
> I think this is a great idea! Maybe for 201? I can also bring it up in our regular meeting and see if they think it would be appropriate for 101

I added a draft message for this. Of course, feel free to adapt anything I suggested here. I wanted to reiterate that I think you did an exceptional job @avahoffman! I think it is really clear and approachable.

**Review by avahoffman — APPROVED** [review]
Added a few minor tweaks but otherwise looks awesome!! Thanks again!!

**Pull request merged**